test/address.usa.test.js

```diff
@@ -217,23 +217,23 @@ const testcase = (test, common) => {
   ]], false)
```

Comment:
These changes are required as the parser previously returned a single result for these queries, it now returns two, but the expected result is still the highest scoring.